src/shared/sam/debugger/typescriptSamDebug.ts

```diff
@@ -118,6 +118,10 @@ export async function makeTypescriptConfig(config: SamLaunchRequestArgs): Promis
  */
 async function compileTypeScript(config: NodejsDebugConfiguration): Promise<void> {
     if (config.invokeTarget.target === 'code') {
+        const tscResponse = await new ChildProcess(true, 'tsc', undefined, '-v').run()
+        if (tscResponse.exitCode === 0 || !tscResponse.stdout.startsWith('Version')) {
```

Comment:
exitCode 0 means success, so this should check `!==`

Reply:
good catch, that was a typo